src/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -203,12 +201,38 @@ export class DateTimePickerComponent implements AfterViewInit, OnDestroy {
         public datepicker: DateTimePickerService,
         @Optional() private _rangeService: DateRangeService,
         @Optional() private _rangeOptions: DateRangeOptions) {
+    }
 
-        datepicker.selected$.pipe(distinctUntilChanged(dateComparator), takeUntil(this._onDestroy))
-            .subscribe(date => this.dateChange.emit(date));
+    ngOnInit(): void {
 
-        datepicker.timezone$.pipe(distinctUntilChanged(timezoneComparator), takeUntil(this._onDestroy))
-            .subscribe((timezone: DateTimePickerTimezone) => this.timezoneChange.emit(timezone));
+        // if there is no defined timezone then try and use the system default, otherwise fallback to
+        // offset of 0, e.g. GMT or user defined equivalent
+        if (!this.timezone) {
+            this.datepicker.timezone$.next(this.datepicker.getCurrentTimezone());
+        }
+
+        this.datepicker.selected$.pipe(distinctUntilChanged(dateComparator), takeUntil(this._onDestroy))
+            .subscribe(date => {
+                if (!this.date && !date || dateComparator(this.datepicker.date$.value, date)) {
+                    return;
+                }
+                this.dateChange.emit(date);
+            });
```

Comment:
What was the reason for adding this?

Reply:
This was just moved down from the constructor, it was emitting when the user was programmatically setting the date.

Reply:
Similar to the other comment, I may be wrong but I don't think this protects against programmatic changes. Fixing that can be done as part of another ticket though, unless it affects this timezone issue.

Reply:
I have raised a ticket for this issue and reverted code that I put in regarding it - https://portal.digitalsafe.net/browse/EL-3820